Autopilot/PathManager/Inc/waypointManager.hpp

```diff
@@ -213,7 +218,7 @@ class WaypointManager {
     //Home base
     _PathData * homeBase;
 
-    // For calculating desired track
+    // For calculating desired heading. This affects the sensitivity of the given desired headings
```

Comment:
why is waypoint manager dealing with heading? Shouldn't it only be dealing with track fed in from sensor fusion?

Reply:
Yeah sorry, wrote that before we switched to using track